### PR TITLE
Implement unit test for jobs API

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: jobs/jobs_test.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	}
+
+}
+
+// configure api you want to test
+func setUpRouter(engine *gin.Engine) {
+
+	engine.GET(GET_ALL_JOBS_API_URL, repo.Careers)
+	engine.GET("/api/careers/:id", repo.CareerById)
+
```
Comment (cp-sumi-k) [inline jobs/jobs_test.go:51]: We need to write test for `send-career-mail` also

File: utils/test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	Url               string
+	Method            string
+	Headers           map[string]interface{}
+	Body              interface{}
+	ResponseCode      int
+	ResponseTypeArray bool
+	ExpectedData      interface{}
+}
+
+func nullString(value string) null.String {
```
Comment (cp-sumi-k) [inline utils/test.go:25]: What is use of this function?

File: jobs/jobs_test.go [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	testRequests = []utils.TestRequest{
+		{
+			GET_ALL_JOBS_API_URL,
+			"GET",
+			nil,
+			nil,
+			http.StatusOK,
+			true,
+			[]interface{}{expectedJobsData()},
+		},
```
Comment (cp-sumi-k) [inline jobs/jobs_test.go:41]: Is this method testing  `/api/careers/:id` api?

File: jobs/jobs_test.go [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+			got = utils.GotArrayData(w, t)
+		} else {
+			got = utils.GotData(w, t)
+		}
+
+		assert.Equal(t, testData.ExpectedData, got)
+
+	}
+	utils.TruncateTables(testDB)
+	utils.PrepareTablesData(testDB)
```
Comment (cp-sumi-k) [inline jobs/jobs_test.go:90]: Shouldn't we have to insert data before running the test?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Comment by jimmy0251** [comment]
Let's also run the test on CI to make sure they are passing!

**Review by jimmy0251 — DISMISSED** [review]

**Review by cp-sumi-k — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**